modules/investigating-kernel-crashes.adoc

```diff
@@ -2,162 +2,37 @@
 //
 // * support/troubleshooting/troubleshooting-operating-system-issues.adoc
 
-:_content-type: PROCEDURE
+:_content-type: CONCEPT
 [id="investigating-kernel-crashes"]
 = Investigating kernel crashes
 
-== Enabling kdump
+The `kdump` service, included in `kexec-tools`, provides a crash-dumping mechanism. You can use this service to save the contents of a system's memory for later analysis.
 
-The `kdump` service, included in `kexec-tools`, provides a crash-dumping mechanism. You can use this service to save the contents of the system's memory for later analysis.
+Certain architectures support kdump in Technology Preview (TP) status, whereas `x86_64` architecture supports kdump in General Availability (GA) status.
 
-The `kdump` service is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+The following table provides details about the support level of kdump for different architectures.
 
-For more information about the support scope of Red Hat Technology Preview
-features, see link:https://access.redhat.com/support/offerings/techpreview/[].
+.Kdump support in {op-system}
+[cols=",^v,^v width="100%",options="header"]
+|===
+|Architecture |Support level
 
-{op-system} ships with `kexec-tools`, but manual configuration is required to enable `kdump`.
+a|
+`x86_64`
+| GA
 
-.Procedure
+a|
+`ARM64`
```

Comment:
Is `ARM64` the preferred way of referring to the ARM architecture?  I've also seen it referenced as `aarch64` and don't know what is the right way to do so in the docs

Reply:
I wasn't too sure about that either but then I saw a few places referring it as arm64 surrounding kdump (such as [this](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/supported-kdump-configurations-and-targets_managing-monitoring-and-updating-the-kernel) doc) so thought it was okay.

Reply:
I think we use `aarch64` more commonly in the OCP docs; defer to @jldohmann to sort it out  :)

Reply:
I checked with our OCP on ARM docs person, and she said `arm64` is the preference, so I'll update to reflect that. Good question!